<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestPart</class>
 <widget class="QWidget" name="RequestPart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="reasonLabel">
          <property name="text">
           <string>请假事由：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTextEdit" name="reasonEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="startTimeLabel">
          <property name="text">
           <string>请假日期：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="endTimeLabel">
          <property name="text">
           <string>结束日期：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="startTimeEdit">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="date">
           <date>
            <year>2020</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDateEdit" name="endTimeEdit">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="date">
           <date>
            <year>2020</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="createButton">
          <property name="text">
           <string>请假</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="listLabel">
        <property name="text">
         <string>待销假列表：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="actualEndTimeLabel">
            <property name="text">
             <string>实际结束日期：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDateEdit" name="actualEndTimeEdit">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
            <property name="date">
             <date>
              <year>2020</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="finishButton">
          <property name="text">
           <string>销假</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
